cms/static/cms/js/modules/cms.modal.js

```diff
@@ -108,8 +108,27 @@
                 });
             },
 
-            // public methods
-            open: function (url, name, breadcrumb) {
+            /**
+             * opens the modal either in an iframe or renders markup
+             *
+             * @method open
+             * @param opts
+             * @param opts.breadcrumb {Array} collection of breadcrumb items
+             * @param opts.html {String} html markup to render
```

Comment:
strictly speaking this one and `url` a optional so should be wrapped with `[...]`